soc/xtensa/intel_adsp/tools/cavstool_server.py

```diff
@@ -11,6 +11,22 @@
 import ctypes
 import mmap
 import argparse
+import socketserver
+import threading
+import netifaces
```

Comment:
I wonder if `netifaces` shouldn't be a part of `requirements-run-test.txt` or so.

Reply:
You concern is right. I will try to add it to the requirements-run-test.txt. thanks! @edersondisouza

Reply:
Submit https://github.com/zephyrproject-rtos/zephyr/pull/46552 for it.